lldb/source/API/SBTarget.cpp

```diff
@@ -1342,7 +1342,9 @@ lldb::SBWatchpoint SBTarget::WatchAddress(lldb::addr_t addr, size_t size,
 
   SBWatchpointOptions options;
   options.SetWatchpointTypeRead(read);
```

Comment:
Shouldn't this check the `read` flag in a similar fashion?

Reply:
It doesn't need to: It passes read bool directly to the Set... method so it's set based on the value of 'read'